tck/cmd/server.go

```diff
@@ -15,12 +24,23 @@ func main() {
 
 	// Initialize the SDK service
 	sdkService := new(methods.SDKService)
+	accountService := new(methods.AccountService)
+	accountService.SetSdkService(sdkService)
 
 	// Create a new RPC server
 	assigner := handler.Map{
-		"setup": handler.New(sdkService.Setup),
-		"reset": handler.New(sdkService.Reset),
+		"setup":                  errorMiddleware(HandleError, handler.New(sdkService.Setup)),
```

Comment:
Looks like we are initialising the server with errorMiddleware's and not endpoints and etc. Would think of renaming this but anyways this is a NIT.

Reply:
Renamed